src/__stories__/utils/Wrapper.tsx

```diff
@@ -2,19 +2,17 @@ import * as React from 'react';
 
 export const Wrapper: React.FunctionComponent<React.HTMLAttributes<HTMLElement>> = ({ children, ...props }) => {
```

Comment:
We should be able to remove it now. I forgot to do it in https://github.com/stoplightio/json-schema-viewer/pull/179/commits/95cac3c2332cf8cb839b5418548a4a9de84d9d56